terraform/modules/hono/variables.tf

```diff
@@ -71,22 +91,62 @@ variable "device_communication_dns_name" {
 
 variable "api_tls_key" {
   type        = string
-  description = "Filename of the ingress tls/server Key File including file extension"
+  description = "Content of the ingress tls/server Key File"
```

Comment:
is api_tls_key the file path or the key?

Reply:
No, it is the content of the key file